libs/langchain/langchain/memory/entity.py

```diff
@@ -153,6 +158,8 @@ class RedisEntityStore(BaseEntityStore):
 
     Entities get a TTL of 1 day by default, and
     that TTL is extended by 3 days every time the entity is read back.
+
+    Must have `redis` and `langchain-community` installed.
```

Comment:
this should be moved to community, no?

Reply:
yea but BaseEntityStore isn't in community yet. could move, just not something we've deprecated yet